lib/cjs/src/Admin/admin-api/admin-layout.d.ts

```diff
@@ -0,0 +1,7 @@
+import { Image } from "../admin-interface/AdminGalleryTypes";
+export declare const deleteImage: (image: Image) => Promise<any>;
+export declare const getAllImages: () => Promise<any>;
+export declare const getCardRefs: () => Promise<any>;
+export declare const addHeroTagToImg: (image: Image) => Promise<any>;
+export declare const removeHeroTagFromImg: (image: Image) => Promise<any>;
+export declare const getHeroImages: () => Promise<any>;
```

Comment:
@robertbolgeo Why are these returning `any`?

I am cool with any if the data is controlled by an outside api. But if you know what it returns you should look at response types and maybe make a file that shows them.

Can be addressed in a future ticket. But curious.

Reply:
I've added a git issue for this problem #236